File: templates/tests-php/BREACH-001.php
<?php

namespace Compliance\Tests;

use PHPUnit\Framework\TestCase;

class DataRetentionTest extends TestCase
{
    public function testEnforceDataRetentionPolicy(): void
    {
        $config = Config::load();
        $retention = $config->dataLifecycle->retention;
        $this->assertGreaterThan(0, $retention);
        $this->assertLessThanOrEqual(365, $retention);
    }
}

File: templates/tests-csharp/BREACH-001.cs
using Xunit;

namespace Compliance.Tests
{
    public class DataRetentionTests
    {
        [Fact]
        public void TestEnforceDataRetentionPolicy()
        {
            var config = Config.Load();
            Assert.True(config.DataLifecycle.Retention > 0);
            Assert.True(config.DataLifecycle.Retention <= 365);
        }
    }
}
